src/ucp/tag/tag_match.h

```diff
@@ -13,6 +13,10 @@
 #include <ucs/sys/compiler_def.h>
 
 
+#define UCP_TAG_MASK_FULL           ((ucp_tag_t)-1)
+#define UCP_TAG_MATCH_HASH_SIZE     945
```

Comment:
will replace with prime number, wanted to create hash table of ~1000 entries which is small enough to fit cahces